src/main/java/com/ldtteam/blockout/controls/ItemIcon.java

```diff
@@ -122,11 +123,11 @@ private void renderGuiItemDecorations(
         if (stack.getCount() != 1)
         {
             String s = String.valueOf(stack.getCount());
-            matrixstack.translate(0.0D, 0.0D, 200.0D);
+            matrixstack.translate(0.0D, 0.0D, 150.0D);
             IRenderTypeBuffer.Impl irendertypebuffer$impl = IRenderTypeBuffer.immediate(Tessellator.getInstance().getBuilder());
             fontRenderer.drawInBatch(s,
-              (float) (19 - 2 - fontRenderer.width(s)),
-              (float) (6 + 3),
+              DEFAULT_ITEMSTACK_SIZEF + 1 - fontRenderer.width(s),
```

Comment:
Please refrain from constanting vanilla inlines

Reply:
What do you mean by that?

Reply:
This code is vanilla inline = code from vanilla which can't be for some reason called directly
Thus it doesn't make sense to put constants in it since you need to changes as easier as possible

Reply:
But (a) you're not going to change it again, as 1.18 has already moved on, and (b) it makes sense to use the constants where they're meaningful, otherwise why have them? and (c) how was I supposed to know that?  There's nothing in the code that indicates that.

Reply:
a) true
b) true except this since then its harder to compare with vanilla
c) true, its probably that old it does not have comment with inline description, but u could guess that from decompiler-codestyle